src/components/Map/scss/MapMarker.scss

```diff
@@ -0,0 +1,10 @@
+/*------------------------------------*\
+    $MAP MARKER
+\*------------------------------------*/
+
+.map-marker {
+  width: 3rem;
+  height: 1rem;
+  background-color: black;
+  color: white;
```

Comment:
why are we moving away from only keeping functional css rules?

Reply:
this should be removed...